docs/reference/configuration.md

```diff
@@ -11,12 +11,14 @@ This document will explain the usage of the different tables.
 For more technical documentation check pixi on [crates.io](https://docs.rs/pixi/latest/pixi/project/manifest/struct.ProjectManifest.html).
 
 !!! tip
-    We also support the `pyproject.toml` file. It has the same structure as the `pixi.toml` file. except that you need to prepend the tables with `tool.pixi` instead of just the table name.
-    For example, the `[project]` table becomes `[tool.pixi.project]`.
-    There are also some small extras that are available in the `pyproject.toml` file, checkout the [pyproject.toml](advanced/pyproject_toml.md) documentation for more information.
+We also support the `pyproject.toml` file. It has the same structure as the `pixi.toml` file. except that you need to prepend the tables with `tool.pixi` instead of just the table name.
+For example, the `[project]` table becomes `[tool.pixi.project]`.
+There are also some small extras that are available in the `pyproject.toml` file, checkout the [pyproject.toml](../advanced/pyproject_toml.md) documentation for more information.
```

Comment:
```suggestion
    We also support the `pyproject.toml` file. It has the same structure as the `pixi.toml` file. except that you need to prepend the tables with `tool.pixi` instead of just the table name.
    For example, the `[project]` table becomes `[tool.pixi.project]`.
    There are also some small extras that are available in the `pyproject.toml` file, checkout the [pyproject.toml](../advanced/pyproject_toml.md) documentation for more information.
```